packages/flask/src/constants/index.ts

```diff
@@ -63,6 +63,7 @@ export const UIConstant = {
     calendar: {
         dayCellPadding: 5,
         dayCellPaddingBorderRadius: 20,
+        daySize: 34,
```

Comment:
I actually didn't want to use the exact size, because
1. it makes impossible to use dynamic font size (i.e. for accessibility)
2. it might break on every font size change

Do you think we can come up with a better idea, how to achieve it without providing specific size? If it's impossible then it's ok, but I really want to think twice about it before merging